### PR TITLE
updated client lib to apiv2

### PR DESCRIPTION
**Pull request by andhess**

### CODE REVIEW

**Comment by philipithomas** [comment]
@andhess Tests failed

**Comment by andhess** [comment]
how do i set env variables in test?

File: staffjoy/resources/schedule_timeclock.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from ..resource import Resource
+
+
+class ScheduleTimeclock(Resource):
+    PATH = "organizations/{organization_id}/locations/{location_id}/roles/{role_id}/schedules/{schedule_id}/timeclocks/"
```
Comment (philipithomas) [inline staffjoy/resources/schedule_timeclock.py:6]: Shouldn't this have a SCHEDULE_ID and include the `timeclock_id` in the path?

Comment (andhess) [inline staffjoy/resources/schedule_timeclock.py:6]: this is a read collection only endpoint to get timeclocks that occur during a schedule

Comment (philipithomas) [inline staffjoy/resources/schedule_timeclock.py:6]: Ok, might be good to add a comment to that effect

**Comment by philipithomas** [comment]
@andhess We use TravisCI, so you can add the env variable to `.travis.yml` if it's public or through the interface if it's secret

**Pull request merged**